pkg/plugins/trivy/plugin_test.go

```diff
@@ -532,9 +532,10 @@ func TestPlugin_Init(t *testing.T) {
 				ResourceVersion: "1",
 			},
 			Data: map[string]string{
-				"trivy.imageRef":                  "ghcr.io/aquasecurity/trivy:0.30.0",
-				"trivy.severity":                  "UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL",
-				"trivy.mode":                      "Standalone",
+				"trivy.imageRef": "ghcr.io/aquasecurity/trivy:0.30.0",
+				"trivy.severity": "UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL",
+				"trivy.mode":     "Standalone",
+				"trivy.additionalVulnerabilityReportFields": "",
```

Comment:
could you please update [trivy docs](https://github.com/aquasecurity/trivy-operator/blob/main/docs/vulnerability-scanning/trivy.md)

Reply:
done